### PR TITLE
add support for specifying custom location for scripts/binaries via setup.py

### PR DESCRIPTION
**Pull request by boegel**

### CODE REVIEW

File: lib/vsc/install/shared_setup.py [hunk 1]
```diff
@@ -1552,11 +1552,20 @@ def search_replace(txt):
...
+
+        Creates [bdist_rpm] section with
             install_requires => requires
             provides => provides
             setup_requires => build_requires
 
+        Creates [metadata] section with
+            description-file => README file
+
+        Creates [install] section if needed,
```
Comment (stdweird) [inline lib/vsc/install/shared_setup.py:1565]: only when building rpm?

Comment (boegel) [reply lib/vsc/install/shared_setup.py:1565]: No, only when `install-scripts` is specified in `setup.py`.

Or I don't understand what you want me to change :)

Comment (stdweird) [reply lib/vsc/install/shared_setup.py:1565]: the is called in a method called build_setup_cfg_for_bdist_rpm, i'm going to assume that only rus when building rpms?

Comment (boegel) [reply lib/vsc/install/shared_setup.py:1565]: Yes, and that's fine. We only need to control the location of the scripts when building an RPM, not when installing via `setup.py install`.

**Pull request merged**